nextflow/conf/03_environment_profiles.config

```diff
@@ -0,0 +1,83 @@
+// *****************
+// PROFILE SETTINGS
+// conda, docker, etc
+// *****************
+
+
+profiles {
+
+   docker {
+    process.container = 'jason-c-kwan/autometa:nfcore'
```

Comment:
I've been struggling with the best way to approach this as well, but haven't thought about it a lot.  
agree with:

> after release of Autometa 2.0 should this be updated to jason-c-kwan/autometa:latest

Reply:
I'd like to keep this for now so people working on the PR won't be confused. As the author I can't request a change so can you request a change on line 10 so that the PR is blocked until this is changed?

Reply:
Will do